docs-source/content/_index.md

```diff
@@ -79,9 +73,9 @@ or suggestions about what features and improvements you would like to see.  We w
 please [visit this site to get an invitation](https://weblogic-slack-inviter.herokuapp.com/).  The invitation email will include
 details of how to access our Slack workspace.  After you are logged in, please come to `#operator` and say, "hello!"
 
-### Recent changes
+### Recent changes and known issues
 
-See [Recent changes]({{< relref "release-notes.md" >}})  for changes to the operator, including any backward incompatible changes.
+See the [Recent Notes]({{< relref "release-notes.md" >}})  for recent changes to the operator and known issues.
```

Comment:
release notes?

Reply:
Yes, we are calling the new recent changes doc, Release Notes. It has the recent changes and known issues. Does that not work for you?